tests/subsys/dfu/dfu_target_mcuboot/pm_config.h

```diff
@@ -10,4 +10,6 @@
 #define PM_S0_ADDRESS 0x8000
 #define PM_S1_ADDRESS 0x15000
 #define PM_MCUBOOT_SECONDARY_SIZE 0x5e000
+#define PM_MCUBOOT_SECONDARY_ADDRESS 0x1d000
+#define PM_MCUBOOT_SECONDARY_DEV_NAME "NRF_FLASH"
```

Comment:
With Upmerge context, this lines conflict with this commit: https://github.com/nrfconnect/sdk-nrf/pull/3569/commits/491e6437a2e30e0568c4491a86a02283bdb85cf9

Reply:
Ok, I can remove it from this PR then, since its coming in the upmerge anyway

Reply:
check now